googlepubsub/json_schemas/message.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
I also noticed your schemas do not allow `patternProperties` with `x-` as the other schemas. Is it intentional, you do not want to allow extensions inside binding?

Reply:
It was intentional at first but I really can't think of a good reason why we shouldn't.  I'll add support.